storybook/storybook-css/src/docs/focus.docs.mdx

```diff
@@ -0,0 +1,16 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/Designrichtlijnen/Focus staat" />
+
+# Focus staat
```

Comment:
This is a rather literal translation. I’d propose ‘focus-indicator’, which highlights the visual aspect of _indicating_ a certain state, not the state itself.